package.json

```diff
@@ -67,6 +63,8 @@
     "run-sequence": "^2.2.0",
     "standard": "^10.0.2",
     "vinyl-paths": "^2.1.0",
+    "webpack": "^4.2.0",
+    "webpack-cli": "^2.0.12",
```

Comment:
why do we need cli?

Reply:
we need it to run  webpack i.e.`webpack --config config/webpack.config.js` in `compile-assets.js`

Reply:
👍